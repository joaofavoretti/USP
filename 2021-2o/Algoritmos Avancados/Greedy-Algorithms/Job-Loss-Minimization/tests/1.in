3
2
5 3
2 1
3
1 10
1 5
1 7
3
10 1
5 1
7 1

