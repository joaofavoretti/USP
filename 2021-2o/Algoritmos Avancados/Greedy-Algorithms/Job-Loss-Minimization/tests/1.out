22
39
39
